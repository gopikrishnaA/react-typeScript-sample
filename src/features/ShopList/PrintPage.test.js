import React from "react";
import { shallow } from "enzyme";
import renderer from 'react-test-renderer';
import { PrintPage } from './PrintPage';


describe("RecipeItem", () => {
    let wrapper;
    const items = [{
        id: 123,
        name: 'test',
        unit: '1',
        amount: 1000
    }]
    beforeEach(() => {
        wrapper = shallow(<PrintPage items={items} customItems={[]} />);
    })

    it("renders correctly", () => {
        const tree = renderer
        .create(wrapper.find('[data-test="table"]'))
        .toJSON();
      expect(tree).toMatchSnapshot();
    })
})