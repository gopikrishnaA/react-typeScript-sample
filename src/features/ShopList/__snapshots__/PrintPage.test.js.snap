// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeItem renders correctly 1`] = `
<table
  data-test="table"
  style={
    Object {
      "fontSize": "20px",
      "margin": "30px",
      "width": "100%",
    }
  }
>
  <thead>
    <tr>
      <th>
        Amount
      </th>
      <th>
        Unit
      </th>
      <th>
        Name
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        1000
      </td>
      <td>
        1
      </td>
      <td>
        test
      </td>
    </tr>
  </tbody>
</table>
`;
